x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -113,12 +116,13 @@ export async function installKibanaAssets(options: {
     logger,
     savedObjectsImporter,
     kibanaAssets: [...indexPatternSavedObjects, ...assetsToInstall],
+    spaceId,
   });
 
   return installedAssets;
 }
 export const deleteKibanaInstalledRefs = async (
-  savedObjectsClient: SavedObjectsClientContract,
+  savedObjectsClient: ISavedObjectsRepository,
```

Comment:
Should we rename all of these param names to `savedObjectsRepo` in all of these files? (at least in Fleet)